build.gradle

```diff
@@ -0,0 +1,120 @@
+    implementation 'io.netty:netty-codec:4.1.94.Final'
+    implementation 'io.netty:netty-common:4.1.94.Final'
+    implementation 'io.netty:netty-handler-proxy:4.1.94.Final'
+    implementation 'io.netty:netty-handler:4.1.94.Final'
+    implementation 'io.netty:netty-resolver-dns-classes-macos:4.1.94.Final'
+    implementation 'io.netty:netty-resolver-dns-native-macos:4.1.94.Final:osx-x86_64'
+    implementation 'io.netty:netty-resolver-dns:4.1.94.Final'
+    implementation 'io.netty:netty-resolver:4.1.94.Final'
+    implementation 'io.netty:netty-tcnative-boringssl-static:2.0.61.Final'
+    implementation 'io.netty:netty-tcnative-classes:2.0.61.Final'
+    implementation 'io.netty:netty-transport-classes-epoll:4.1.94.Final'
+    implementation 'io.netty:netty-transport-classes-kqueue:4.1.94.Final'
+    implementation 'io.netty:netty-transport-native-epoll:4.1.94.Final:linux-x86_64'
+    implementation 'io.netty:netty-transport-native-kqueue:4.1.94.Final:osx-x86_64'
+    implementation 'io.netty:netty-transport-native-unix-common:4.1.94.Final'
+    implementation 'io.netty:netty-transport:4.1.94.Final'
+    implementation 'io.projectreactor.netty:reactor-netty-core:1.0.33'
+    implementation 'io.projectreactor.netty:reactor-netty-http:1.0.33'
+    implementation 'io.projectreactor:reactor-core:3.4.32'
+    implementation 'io.vavr:vavr-match:0.10.2'
+    implementation 'io.vavr:vavr:0.10.2'
+    implementation 'net.java.dev.jna:jna-platform:5.13.0'
+    implementation 'net.java.dev.jna:jna:5.13.0'
+    implementation 'net.minidev:accessors-smart:2.4.9'
+    implementation 'net.minidev:json-smart:2.4.10'
+    implementation 'org.apache.commons:commons-lang3:3.12.0'
+    implementation 'org.apache.commons:commons-text:1.10.0'
+    implementation 'org.apache.httpcomponents:httpclient:4.5.14'
+    implementation 'org.apache.httpcomponents:httpcore:4.4.16'
+    implementation 'org.codehaus.woodstox:stax2-api:4.2.1'
+    implementation 'org.jetbrains:annotations:22.0.0'
+    implementation 'org.ow2.asm:asm:9.3'
+    implementation 'org.reactivestreams:reactive-streams:1.0.4'
+    implementation 'org.slf4j:slf4j-api:1.8.0-beta4'
+    implementation 'org.slf4j:slf4j-simple:1.8.0-beta4'
+}
+
+task generateGemJarRequiresFile {
+    doLast {
+        File jars_file = file('lib/logstash-output-kusto_jars.rb')
+        jars_file.newWriter().withWriter { w ->
+            w << "# AUTOGENERATED BY THE GRADLE SCRIPT. DO NOT EDIT.\n\n"
+            w << "require \'jar_dependencies\'\n"
+            configurations.runtimeClasspath.allDependencies.each {
+                w << "require_jar(\'${it.group}\', \'${it.name}\', \'${it.version}\')\n"
+            }
+            // w << "require_jar(\'${project.group}\', \'${project.name}\', \'${project.version}\')\n"
+        }
+    }
+}
+
+task vendor {
+    doLast {
+        String vendorPathPrefix = "lib"
+        configurations.runtimeClasspath.allDependencies.each { dep ->
```

Comment:
Add comments to the file explaining what everything does

Reply:
Add some comments as well for these steps